Demo/Demo/Models/Order.swift

```diff
@@ -1,5 +1,39 @@
 struct Order: Codable {
 
+    enum CodingKeys: String, CodingKey {
+        case id
+        case status
+        case paymentSource = "payment_source"
+    }
```

Comment:
We don't need custom coding keys if we use the `JSONEncoder.convertToSnakeCase` setting ([see Apple docs](https://developer.apple.com/documentation/foundation/jsonencoder/keyencodingstrategy/converttosnakecase))

Reply:
I ran into trouble when decoding when I removed the coding keys.